js/geo/transform.js

```diff
@@ -375,7 +394,8 @@ Transform.prototype = {
                 y2 !== undefined ? y2 : this.y));
         }
 
-        this._constraining = false;
+        this._unmodified = this._constraining.unmodified;
+        this._constraining = undefined;
```

Comment:
We prevent `_constrain` recursion in its first line, so saving `_unmodified` state inside `_constraining` seems unnecessary. I'm just trying to make sure we don't add any unnecessary code, because little things like this tend to pile up.